package-lock.json

```diff
@@ -3898,9 +3897,6 @@
       "version": "4.0.0",
       "resolved": "https://registry.npmjs.org/jsonfile/-/jsonfile-4.0.0.tgz",
       "integrity": "sha1-h3Gq4HmbZAdrdmQPygWPnBDjPss=",
-      "dependencies": {
-        "graceful-fs": "^4.1.6"
-      },
```

Comment:
@wadamek65 As I guessed, it's an OS-related thing. Let's leave it like that, but keep an eye on it.